lib/src/int.dart

```diff
@@ -0,0 +1,29 @@
+part of dartx;
+
+extension IntSuffix<T extends int> on T {
+  /// Returns a string adding a proper suffix on any integer
+  ///
+  /// ```dart
+  /// 101.addSuffix(); // 101st
+  ///
+  /// 999218.addSuffix(); // 999218th
+  /// ```
+  String addSuffix() {
```

Comment:
I'd call it `ordinal()`

Reply:
Aaahh Thanks. I was exactly looking for this very word 😂
yes ordinal is good one
Thanks